tests/test_metadata.py

```diff
@@ -125,6 +126,9 @@ def _get_container_label_prefix(
         (pg_container, "postgres", ImageType.APPLICATION)
         for pg_container in POSTGRESQL_CONTAINERS
     ]
+    + [
+        (DISTRIBUTION_CONTAINER, "distribution", ImageType.APPLICATION),
```

Comment:
The image is called registry.
```suggestion
        (DISTRIBUTION_CONTAINER, "registry", ImageType.APPLICATION),
```

Reply:
Yes, I check the changes first locally before waiting for the slow CI.